.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Crate Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+
+### Feature Categories:
+
+
+- [ ] Express
+- [ ] React
+- [ ] Styling/CSS
+- [ ] Documentation
+- [ ] Other: 
+
+**Describe the feature/Bug your gonna work on**
+
+
+
+**Additional context**
+Add any other context or screenshots about the feature.
```

Comment:
Screenshot are great and GH lets you just drag and drop images into the issue

Reply:
No way ! That is SO cool. The things you learn on group projects.